### DISCUSSION TITLE
Help me understand the zinc dataset

### DISCUSSION THREAD

**Discussion by tonydavis629** [pyg-team/pytorch_geometric#6273] [Q&A]
In the docstring, it describes Zinc as having 249,456 graphs, 23.2 nodes, 49.8 edges, 1 feature and 1 class. When I load the dataset, I get a very different shape. 
```
from torch_geometric.datasets import ZINC
dataset = ZINC(path)
print(dataset.data)
>>> Data(x=[5094322, 1], edge_index=[2, 10959434], edge_attr=[10959434], y=[220011])
```
What is going on here? It looks like every sample is getting concatenated into the Data object. I would expect `Data.x` to be of the type `List[Data]` of length 249,456, with each Data object representing a molecular graph. Is this expected behavior, or is this a bug?

**Comment by rusty1s** [comment]
`dataset.data` is our internal storage format. You can access individual elements via `dataset[i]`.
